### PR TITLE
fix(dedicated): user rights text

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-5997
| License          | BSD 3-Clause

## Description

Fix text for essentials and premier offers

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/edit/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
   "dedicatedCloud_USER_right_vmNetwork_noAccess": "Aucun accès",
   "dedicatedCloud_USER_right_vmNetwork_READONLY": "Accès en lecture seule",
   "dedicatedCloud_USER_right_vmNetwork_readonly": "Accès en lecture seule",
-  "dedicatedCloud_USER_right_vlan": "Accès au V(x)Lans",
-  "dedicatedCloud_USER_right_vlan_help_EPCC": "Droits de management sur la partie réseau privé (VxLan pour les Dedicated Cloud et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
-  "dedicatedCloud_USER_right_vlan_help_MBM": "Droits de management sur la partie réseau privé (VxLan pour les Essentials et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
+  "dedicatedCloud_USER_right_vlan_EPCC": "Accès au V(x)Lans",
+  "dedicatedCloud_USER_right_vlan_MBM": "Accès au VLans",
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/edit/translations/Messages_fr_FR.json:38]:
```suggestion
  "dedicatedCloud_USER_right_vlan_MBM": "Accès au VLan",
```
(same for: https://github.com/ovh/manager/pull/4041/files#diff-443ebe31bc9b786d3fd0ce07fb7383fc0a6deaad4fb729ab244801ddb14efce8R41 or all occurences of `au` + `VLan`)

Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/edit/translations/Messages_fr_FR.json]: Thanks a lot @JeremyDec . The sentence has been corrected as `Accès aux VLANs`

**Review by radireddy — APPROVED** [review]

**Review by marie-j — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/edit/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -34,9 +34,9 @@
   "dedicatedCloud_USER_right_vmNetwork_noAccess": "Aucun accès",
   "dedicatedCloud_USER_right_vmNetwork_READONLY": "Accès en lecture seule",
   "dedicatedCloud_USER_right_vmNetwork_readonly": "Accès en lecture seule",
-  "dedicatedCloud_USER_right_vlan": "Accès au V(x)Lans",
-  "dedicatedCloud_USER_right_vlan_help_EPCC": "Droits de management sur la partie réseau privé (VxLan pour les Dedicated Cloud et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
-  "dedicatedCloud_USER_right_vlan_help_MBM": "Droits de management sur la partie réseau privé (VxLan pour les Essentials et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
+  "dedicatedCloud_USER_right_vlan_EPCC": "Accès au V(x)Lans",
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/edit/translations/Messages_fr_FR.json:37]: @ganeshkumar1989 same thing here, this should be
```suggestion
  "dedicatedCloud_USER_right_vlan_EPCC": "Accès aux V(x)Lans",
```

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -1,56 +1,57 @@
...
+  "dedicatedCloud_USER_right_vlan_noAccess": "Aucun accès",
+  "dedicatedCloud_USER_right_vlan_READONLY": "Accès en lecture seule",
+  "dedicatedCloud_USER_right_vlan_readonly": "Accès en lecture seule",
+  "dedicatedCloud_USER_right_vlan_MANAGER": "Permet de gérer les port groups du switch virtuel (création, modification, suppression). Seulement pour les SDDC et les packs Hosted Private Cloud Premier.",
+  "dedicatedCloud_USER_right_vlan_manager": "Permet de gérer les port groups du switch virtuel (création, modification, suppression). Seulement pour les SDDC et les packs Hosted Private Cloud Premier.",
+  "dedicatedCloud_USER_right_resources": "Ajout de ressource",
+  "dedicatedCloud_USER_right_resources_help": "Droit d'ajouter des ressources supplémentaires via le plugin OVH dans vSphere Client",
+  "dedicatedCloud_USER_resource": "Ajout de ressource",
+  "dedicatedCloud_USER_VM_NETWORK_ROLE": "Accès au vmNetwork",
+  "dedicatedCloud_USER_NETWORK_ROLE_EPCC": "Accès au V(x)Lans",
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/translations/Messages_fr_FR.json:55]: Same

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -1,56 +1,57 @@
...
+  "dedicatedCloud_USER_right_vsphere_readwrite": "Accès en écriture",
+  "dedicatedCloud_USER_right_vmNetwork": "Accès au vmNetwork",
+  "dedicatedCloud_USER_right_vmNetwork_help": "Droits de management sur la partie réseau public (appelé VM Network dans l'interface vSphere)",
+  "dedicatedCloud_USER_right_vmNetwork_ADMIN": "Permet de configurer des VMs sur le réseau public",
+  "dedicatedCloud_USER_right_vmNetwork_admin": "Permet de configurer des VMs sur le réseau public",
+  "dedicatedCloud_USER_right_vmNetwork_NO_ACCESS": "Aucun accès",
+  "dedicatedCloud_USER_right_vmNetwork_noAccess": "Aucun accès",
+  "dedicatedCloud_USER_right_vmNetwork_READONLY": "Accès en lecture seule",
+  "dedicatedCloud_USER_right_vmNetwork_readonly": "Accès en lecture seule",
+  "dedicatedCloud_USER_right_vlan": "Accès au V(x)Lans",
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/translations/Messages_fr_FR.json:41]: Same

**Pull request merged**